modules/nf-core/universc/README.md

```diff
@@ -0,0 +1,51 @@
+# UniverSC
+
+## Single-cell processing across technologies
+
+UniverSC is an open-source single-cell pipeline that runs across platforms on various technologies.
+
+## Maintainers
+
+Tom Kelly (RIKEN, IMS)
+
+Kai Battenberg (RIKEN CSRS/IMS)
+
+Contact: <first name>.<family name>[at]riken.jp
+
+## Implementation
+
+This container runs Cell Ranger v3.0.2 installed from source on MIT License on GitHub with
```

Comment:
Thank you, I wasn't sure where to put documentation for this. Note the licensing for the Cell Ranger 3.0.2 release is MIT whereas UniverSC itself is on GPL3.0. Both permit modification and use without 10X EULA restrictions.

It was difficult to install open-source dependencies for Cell Ranger and update the source code for compatibility with new versions. I hope this will be beneficial to the research community, hence my efforts to bundle it into nf-core.